website/docs/components/pagination/partials/code/component-api.md

```diff
@@ -1,20 +1,161 @@
+</C.Property>
+<C.Property @name="queryFunction" @type="function">
+A function that returns an object that can be provided as `query` argument for the routing, and that is passed down to the to the child components together with the other routing parameters. The function receives as argument one of two possible values: `prev` / `next`.
+</C.Property>
+<C.Property @name="isDisabledPrev/isDisabledNext" @type="boolean" @values={{array "true" "false" }} @default="false">
+Used to disable the "prev" or "next" controls. Notice: when the control is disabled, it's always rendered as an HTML `<button>` element.
+</C.Property>
+<C.Property @name="showLabels" @type="boolean" @values={{array "true" "false" }} @default="true">
+Used to control the visibility of the "prev/next" text labels.
+</C.Property>
+<C.Property @name="onPageChange" @type="function">
+Callback function invoked (if provided) when a "prev" or "next" control is clicked. The function receives as argument one of two possible values: `prev` / `next`
+</C.Property>
+<C.Property @name="...attributes">
+This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+</C.Property>
+</Doc::ComponentApi>
+
+### Base elements
+
+#### Pagination::Info
+
+<Doc::ComponentApi as |C|>
+<C.Property @name="itemsRangeStart" @required="true" @type="string|number">
+The "start" value of the range in the informational text.
+</C.Property>
+<C.Property @name="itemsRangeEnd" @required="true" @type="string|number">
+The "end" value of the range in the informational text.
+</C.Property>
+<C.Property @name="totalItems" @required="true" @type="string|number">
+The "out of" total items in the informational text. It's not required if `showTotalItems` is set to `false`.
+</C.Property>
+<C.Property @name="showTotalItems" @type="boolean" @values={{array "true" "false" }} @default="true">
+Controls the visibility of the total items in the informational text.
+</C.Property>
+</Doc::ComponentApi>
+
+#### Pagination::Nav::Arrow
+
+<Doc::ComponentApi as |C|>
+<C.Property @name="direction" @required="true" @type="enum" @values={{array "prev" "next"}}>
+Sets the "direction" of the icon and label in the control.
+</C.Property>
+<C.Property @name="route/query/model/models/replace">
+These are the parameters that are passed down as arguments to the `Hds::Interactive` component (used internally). For more details about how this low-level component works please refer to [its documentation page](/utilities/interactive/).
+</C.Property>
+<C.Property @name="disabled" @type="boolean" @values={{array "true" "false" }} @default="false">
+If the control is disabled. Notice: when the control is disabled, it's always rendered as an HTML `Button` element.
```

Comment:
```suggestion
If the control is disabled. Notice: when the control is disabled, it’s always rendered as an HTML `Button` element.
```

Reply:
@KristinLBradley what do you think of something like 

> Sets the control as `disabled`.

instead?